README.md

```diff
@@ -34,35 +32,35 @@ There are two projects in this sample.  Each needs to be separately registered i
-2.On the top bar, click on your account and under the **Directory** list, choose the Active Directory tenant where you wish to register your application.
-2.Click on **More Services** in the left hand nav, and choose **Azure Active Directory**.
-3.Click on **App registrations** and choose **Add**.
-4.Enter a friendly name for the application, for example 'TodoListDaemonWithCert' and select 'Web Application and/or Web API' as the Application Type. 
-5.Since this application is a daemon and not a web application, it doesn't have a sign-in URL or app ID URI.  For these two fields, enter "http://TodoListDaemonWithCert". Click on **Create** to create the application.
-6.While still in the Azure portal, choose your application, click on **Settings** and choose **Properties**.
-7.Find the Application ID value and copy it to the clipboard.
+1. Sign in to the [Azure portal](https://portal.azure.com).
+2. On the top bar, click on your account and under the **Directory** list, choose the Active Directory tenant where you wish to register your application.
+2. Click on **More Services** in the left hand nav, and choose **Azure Active Directory**.
+3. Click on **App registrations** and choose **Add**.
+4. Enter a friendly name for the application, for example 'TodoListDaemonWithCert' and select *'Web Application and/or Web API'* as the Application Type (even if here we have a console application). 
+5. Since this application is a daemon and not a web application, it doesn't have a sign-in URL or app ID URI.  For these two fields, enter "http://TodoListDaemonWithCert". Click on **Create** to create the application.
+6. While still in the Azure portal, choose your application, click on **Settings** and choose **Properties**.
+7. Find the Application ID value and copy it to the clipboard.
 
 
 #### Create a self-signed certificate
 
-To complete this step you will need the makecert.exe and pvk2pfx.exe utilities, which is included in the Windows SDK.  You may have installed the Windows SDK with Visual Studio 2013, in which case you will find it if you search your computer for makecert.exe.  If not, you can download the SDK [here](http://msdn.microsoft.com/en-US/windows/desktop/aa904949).  You only need to install the Windows Software Development Kit part of the SDK.  You can find more information about the makecert.exe utility [here](http://msdn.microsoft.com/en-us/library/bfsktky3(v=vs.110).aspx).
+To complete this step you will need the makecert.exe and, which is included in the Windows SDK.  You may have installed the Windows SDK with Visual Studio, in which case you will find it if you search your computer for makecert.exe.  If not, you can download the SDK [here](http://msdn.microsoft.com/en-US/windows/desktop/aa904949).  You only need to install the Windows Software Development Kit part of the SDK.  You can find more information about the makecert.exe utility [here](http://msdn.microsoft.com/en-us/library/bfsktky3(v=vs.110).aspx).

```

Comment:
This sample requires Visual Studio, so I see little point in detailing how to install makecert. It would be worth mentioning that you need to start a dev command prompt.